apps/rex-api/src/v1/auth/auth-controller.ts

```diff
@@ -51,4 +50,38 @@ export class AuthController {
 				.send({ ...this.baseTransaction, message: this.somethingWentWrong });
 		}
 	};
+
+	public readonly isDuplicateUserName = async ({ query }: Readonly<Request>, response: Readonly<Response>) => {
```

Comment:
Create a follow up task to throttle open connections that are unsecured endpoints.